test/.clang-format

```diff
@@ -1,11 +1,10 @@
----
 Language:        Cpp
 # BasedOnStyle:  Google
```

Comment:
We should probably take this file off from the PR because #2407 was closed.

Reply:
I am not sure about let umar respond on that.

Reply:
Thanks for reverting the changes - for some reason I was thinking that the clang format file wasn't in master yet.